src/signals/incident-management/components/RadioButtonList/index.js

```diff
@@ -47,10 +48,10 @@ const RadioButtonList = ({
   }
 
   return (
-    <FilterGroup id={id}>
+    <FilterGroup id={id} {...rest} >
       {title && <Label data-testid="radioButtonListTitle" as="span" isGroupHeader={false} label={title} />}
 
-      <StyledRadioGroup name={groupName} disabled={disabled} error={error}>
+      <StyledRadioGroup name={groupName} disabled={disabled} error={error} role="radiogroup">
```

Comment:
If this is the radio group, shouldn't the id be set to this element, and the label have a 'htmlFor' property pointing to the `StyledRadioGroup`?